config/environments/development.rb

```diff
@@ -35,6 +35,10 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.delivery_method = :smtp
```

Comment:
if there's styling needed we should use mailcatcher or some other service to be able to preview the emails in development

Reply:
There is no styling needed. But I was using mailcatcher, these lines are the needed setup :)